### PR TITLE
Added yarn model

### PR DESCRIPTION
**Pull request by AmandaPerino**
Please check if my yarn model is correct.

### CODE REVIEW

**Review by PiotrekCodes — APPROVED** [review]

**Review by zynolia — APPROVED** [review]

**Review by DariBerrie — COMMENTED** [review]
Should we add some validations for yarn :name?

File: app/models/yarn.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Yarn < ApplicationRecord
+  belongs_to :user
```
Comment (DariBerrie) [inline app/models/yarn.rb:2]: validates :name, presence: true ?

Comment (AmandaPerino) [reply app/models/yarn.rb:2]: Done!

**Review by zynolia — APPROVED** [review]
Looks great!

**Review by DariBerrie — APPROVED** [review]
Looks good to me 👍

**Comment by DariBerrie** [comment]
✅

**Pull request merged**